VTEX - Logistics API.json

```diff
@@ -53,7 +53,643 @@
+                                                        "description": "Sellers' names."
+                                                    }
+                                                }
+                                            }
+                                        },
+                                        "processingStatus": {
+                                            "type": "object",
+                                            "description": "Shipping policy processing status.",
+                                            "properties": {
+                                                "status": {
+                                                    "type": "integer",
+                                                    "description": "Shipping policy processing status code."
+                                                },
+                                                "errorMessage": {
+                                                    "type": "string",
+                                                    "description": "Error message, when applicable.",
+                                                    "nullable": true
+                                                },
+                                                "errorsMetadata": {
+                                                    "type": "string",
+                                                    "description": "Error metadata, when applicable.",
+                                                    "nullable": true
+                                                }
+                                            }
+                                        },
+                                        "deliveryChannel": {
+                                            "type": "string",
+                                            "description": "If the shipping type is `delivery` or `pickup`."
+                                        },
+                                        "calculationType": {
+                                            "type": "integer",
+                                            "description": "Shipping policy calculation type."
+                                        },
+                                        "isActive": {
+                                            "type": "boolean",
+                                            "description": "If the shipping policy is active (`true`) or not (`false`)." 
+                                        },
+                                        "lastIndexedAt": {
+                                            "type": "string",
+                                            "description": "Date and time since the last [indexing](https://help.vtex.com/en/tutorial/understanding-how-indexation-works), in [ISO 8601 time zone offset format](https://learn.microsoft.com/en-us/rest/api/storageservices/formatting-datetime-values), as in `YYYY-MM-DD hh:mm:ss[.nnnnnnn]+hh:mm`.",
```

Comment:
```suggestion
                                            "description": "Date and time since the last [indexing](https://help.vtex.com/en/tutorial/understanding-how-indexation-works--tutorials_256), in [ISO 8601 time zone offset format](https://learn.microsoft.com/en-us/rest/api/storageservices/formatting-datetime-values), as in `YYYY-MM-DD hh:mm:ss[.nnnnnnn]+hh:mm`.",
```